qemu/tests/cfg/create_large_raw_img.cfg

```diff
@@ -0,0 +1,18 @@
+- create_large_raw_img:
+    only raw
+    virt_test_type = qemu
+    type = create_large_raw_img
+    start_vm = no
+    create_image = no
+    images = "large"
+    image_name_large = "images/large"
```

Comment:
`image_name_large = "large"`, no need to create a subdirectory under the mount dir.